.gitignore

```diff
@@ -13,6 +13,8 @@ build/
 build-eclipse/
```

Comment:
I added the `screenshots/` directory so that screenshots taken for failing UITests will not accidentally be checked in.